src/helpers.js

```diff
@@ -89,7 +89,7 @@ function noteToFreq(note) {
   var wholeNotes = { A: 21, B: 23, C: 24, D: 26, E: 28, F: 29, G: 31 };
   var value = wholeNotes[note[0].toUpperCase()];
   var octave = ~~note.slice(-1);
-  value += 12 * (octave - 1);
+  value += value < 24 ? 12 * octave : 12 * (octave - 1);
```

Comment:
this is a bit confusing to read, compared to changing the values of the first two notes in the `wholeNotes` dictionary in this other PR https://github.com/processing/p5.js-sound/pull/605

`var wholeNotes = { A: 33, B: 35, C: 24, D: 26, E: 28, F: 29, G: 31 };`

Reply:
Yeah I agree :)